azure/firewall/outputs.tf

```diff
@@ -15,5 +15,5 @@ output "public_ip" {
 
 output "private_ip" {
   description = "The private IP Address of the firewall"
-  value       = one(azurerm_firewall.main.ip_configuration.*.private_ip_address)
+  value       = one(azurerm_firewall.main.ip_configuration[*].private_ip_address)
```

Comment:
Not an issue and fine as it is but I'm curious why this was changed? It should mean the same thing?

Reply:
tflint suggested the change so I went with it